app/src/main/kotlin/io/homeassistant/companion/android/settings/sensor/views/SensorListView.kt

```diff
@@ -42,24 +36,15 @@ fun SensorListView(
                 key = manager.id(),
             ) {
                 if (currentSensors.any()) {
-                    Column(
+                    Box(
                         modifier = Modifier
                             .background(MaterialTheme.colors.background)
                             .fillMaxWidth(),
                     ) {
-                        Row(
-                            modifier = Modifier
-                                .height(48.dp)
-                                .padding(start = 72.dp, end = 16.dp),
-                            verticalAlignment = Alignment.CenterVertically,
-                        ) {
-                            Text(
-                                text = stringResource(manager.name),
-                                style = MaterialTheme.typography.body2,
-                                fontWeight = FontWeight.Bold,
-                                color = MaterialTheme.colors.primary,
-                            )
-                        }
+                        SettingsSubheader(
+                            text = stringResource(manager.name),
+                            paddingForIcon = true,
```

Comment:
Maybe not relevant but 

<img width="256" height="226" alt="Image" src="https://github.com/user-attachments/assets/3b3ff3a5-f7df-42ee-8837-a152ec75f1d1" />

Don't we want the icon to be display and not being cut by `App sensors`?

Reply:
Not related indeed

The header is sticky, wouldn't it be weird to have the icon scroll behind but the row title/subtitle obscured?